### PR TITLE
my bar page

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Screenshot 2022-09-02 at 15 19 09

File: app/views/components/_dashboard.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<div class="dashboard">
+  <div class="dashboard-grocery">
+    <h3>You are missing only:</h3>
+    <ul class="grocery-list">
+      <% grocery_list.each do |item| %>
+        <li class="grocery-list-item"><%= item.name.capitalize %></li>
```
Comment (A-drienne) [inline app/views/components/_dashboard.html.erb:6]: item.name.titleize works better as it capitalizes both words. name.capitalize only capitalizes the first word

File: app/views/components/_dashboard.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      <% end %>
+    </ul>
+  </div>
+  <div class="dashboard-saved-cocktails">
+    <h3>Your selected cocktail</h3>
+    <ul class="saved-cocktails-list">
+      <% saved_cocktails.each do |saved_cocktail| %>
+        <li class="saved-cocktails-list-item"><%= saved_cocktail.cocktail.name.capitalize %></li>
```
Comment (A-drienne) [inline app/views/components/_dashboard.html.erb:14]: same here:     cocktail.name.titleize

**Review by A-drienne — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
hey @A-drienne, let's titlelize from Monday on :) 
Thx for understanding

**Pull request merged**